dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/BufferManagerImpl.java

```diff
@@ -30,23 +32,32 @@
     private Buffer<Object> ringBuffer;
     private static final ReentrantReadWriteLock RERWLOCK = new ReentrantReadWriteLock(true);
     private Set<SynchronousHandler> synchronousHandlerSet = new HashSet<SynchronousHandler>();
-    private int capacity;
+    private final int capacity;
 
     private final String sourceId;
     /* Map to keep track of the next event for a handler */
     private final ConcurrentHashMap<String, HandlerStats> handlerEventMap = new ConcurrentHashMap<String, HandlerStats>();
+    private volatile static List<BufferManager> bufferManagerList= Collections.synchronizedList(new ArrayList<BufferManager>());
```

Comment:
now that this is only used with the RERWLOCK write lock I think this could be just...
`private static List<BufferManager> bufferManagerList= new ArrayList<BufferManager>();`